lib/bundler/cli/install.rb

```diff
@@ -168,10 +167,10 @@ def normalize_groups
     end
 
     def normalize_settings
-      Bundler.settings.set_command_option :path, nil if options[:system]
-      Bundler.settings.set_command_option :path, "vendor/bundle" if options[:deployment]
-      Bundler.settings.set_command_option_if_given :path, options["path"]
-      Bundler.settings.set_command_option :path, "bundle" if options["standalone"] && Bundler.settings[:path].nil?
+      CLI::Common.set_path nil if options[:system]
+      CLI::Common.set_path "vendor/bundle" if options[:deployment]
+      CLI::Common.set_path options["path"], :if_given
```

Comment:
Can this not just be `true`?

Reply:
`:if_given` tells you what the argument is, since we can't use named parameters yet